### ISSUE TITLE
Sсreen Reader: Many Pages: Various buttons are announced without a role

### ISSUE THREAD

**Issue by olenamihno** [Expensify/App#77566]
## Action Performed:
Precondition: The User should be logged in

1. Using iOS VoiceOver, open the Expensify app
2. Swipe to the 'Account' button at the bottom right corner of the screen and double-tap to activate
3. Swipe between various buttons (Profile, Wallet, Preferences, etc.)
4. Note how the controls are announced.

Other Occurrences:
1. On Status: the button 'Clear status'
2. On Wallet: Add bank account, Enable wallet
3. On Workspaces: New workspace
4. On Troubleshoot
5. On Create Workspace - Options

## Expected Result:
Each option should be announced with a button role.

## Actual Result:
Various buttons are announced without a role, e.g.:
- Profile, Wallet, Preferences etc

## Area issue was found in:
Profile

## Failed WCAG checkpoints
WCAG 4.1.2 Name, Role, Value
https://www.w3.org/WAI/WCAG22/Understanding/name-role-value

## User impact:
Providing role, state, and value information on all user interface components enables compatibility with assistive technology, such as screen readers, screen magnifiers, and speech recognition software, used by people with disabilities.

## Suggested resolution:
Explicitly set the accessibilityRole prop to "button" on each interactive button component.
https://reactnative.dev/docs/button

## Environment:
Apple iPhone 11 - iOS 17.5.1, H0T Mobile - Native (No mobile browser)

## Screenshots/Videos

https://github.com/user-attachments/assets/f1b96c52-4582-4c19-b8e2-83c8de974c7c

<img width="828" height="1792" alt="Image" src="https://github.com/user-attachments/assets/336cbb63-d101-436b-a265-6337bb6a563f" />

**Version Number:** Production
**Reproducible in staging?:** NA
**Reproducible in production?:** Yes
**Email or phone of affected tester (no customers):** No
**Logs:** https://stackoverflow.com/c/expensify/questions/4856  NA
**Notes/Photos/Videos:** Any additional supporting documentation  Yes
**Expensify/Expensify Issue URL:** https://platform.applause.com/company/12102/products/36628/community-issues/7006027
**Issue reported by:** NA
**Slack conversation:** NA
